kitchen.recipes.yml

```diff
@@ -17,16 +17,14 @@ suites:
     run_list:
       # runs the recipe to test, along with the recipes it depends on
       - recipe[aws-parallelcluster::test_dummy]
-      - recipe[aws-parallelcluster::example_recipe]
+      - recipe[aws-parallelcluster-install::sudoers]
       #- recipe[aws-parallelcluster-install::base]
```

Comment:
Nitpick: could you remove the commented line?

Reply:
yes